massgov/mayflower
# DP-20659 Caspio template

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description


- Set up a template to render Caspio content as a figure variation template.
- Add `aria-label` to `<figure>` to add a semantic title to figure.
- Hide the optional title with `<span>` in `<figure>` the duplicate information to `aria-label`, which is the duplicate information to `aria-label` and isn't presented semantically, from screen readers.
- Adjust the optional title display setting in all figure templates.
- Add a Caspio component sample to 05-pages/information-details.
 
## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-20659)
- Drupal PR:  https://github.com/massgov/openmass/pull/557

## Steps to Test


1.  Go to all figure templates under (packages/patternlab/styleguide/source/_patterns/01-atoms/09-media/) and check for proper rendering of the components.

2. Go to **05-pages/information-details** and find a sample of Caspio component.

3. Pull this branch in Drupal (DP-20500) and add a Caspio content in an info. details page and check for its rendering.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
